handlers/main.yml

```
@@ -4,3 +4,4 @@
     name: sshd
     state: reloaded
   become: true
+  when: ansible_fact["os_family"] != "Darwin"
```

Thanks for your pull request! Should this be `ansible_facts` instead of `ansible_fact`?

```suggestion
  when: ansible_facts["os_family"] != "Darwin"
```

---

Yes! Sorry, I've tested the code directly in my local Ansible deployment and then when I copied over to the git repo I've forgot an "s".